src/theme/badges/CloudAvailableBadge/index.js

```diff
@@ -15,7 +15,7 @@ const Icon = () => {
 const CloudAvailableBadge = () => {
     return (
         <div classes className={styles.cloudBadge}>
-            <Icon />{'Only available in ClickHouse Cloud'}
+            <Icon />{'ClickHouse Cloud only'}
```

Comment:
Should that not read "Only has an effect in ClickHouse Cloud?"

Reply:
Well, I guess "ClickHouse Cloud only" is also okay (the badge applies to lots of stuff, not only settings)